maxim/tests/test_test_runs.py

```
@@ -14,17 +14,20 @@
     YieldedOutput,
+    YieldedOutputMeta,
+    YieldedOutputCost,
+    YieldedOutputTokenUsage,
 )
 from maxim.models.evaluator import (
     PassFailCriteriaForTestrunOverall,
     PassFailCriteriaOnEachEntry,
 )
 
-with open(str(f"{os.getcwd()}/libs/maxim-py/maxim/tests/testConfig.json")) as f:
+with open(str(f"{os.getcwd()}/maxim/tests/testConfig.json")) as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)